docs/concepts/customization.md

```diff
@@ -235,7 +235,7 @@ EPSG6933 = morecantile.TileMatrixSet.custom(
 )
 
 # 2. Register TMS
-morecantile.tms.register(custom_tms.EPSG6933)
+morecantile.tms.register(custom_tms=EPSG6933)
```

Comment:
this changed recently. To register a custom TMS you need to do
```python
from morecantile import tms

tms = tms.register(EPSG6933)
```

https://github.com/developmentseed/titiler/blob/09448211e95f406526701a31064a659460e4bbb7/titiler/dependencies.py#L24

Reply:
Note: this changed in morecantile 2.0